### ISSUE TITLE
Cleanup

### ISSUE THREAD

**Issue by pedershk** [twright-msft/mssql-charts#1]
Hi.

Thanks for this, which I found from the Build session. 

Will this eventually end up on the "official" kubernetes charts repo - or is that only for OSS software? Also - the documentation from the Postgres chart seems a bit off ;)

**Comment by twright-msft** [comment]
Yeah, if approved in the next couple of weeks during planning we will be building an _official_ helm chart and publishing it.  For now this is just a PoC for me to drive awareness, understanding, and commitment to do this on the engineering team.
Yeah, I obviously borrowed the Postgres one as a starting point. :)

**Comment by pedershk** [comment]
Hello again.

Now that you've hit RTM - will there be official guidance/helm chart on how to do this? A bit scary to try and run this, even if only for QA/Acceptance Testing, without more details available on how to properly parameterize, recommendations on volume setup and so on.. :)

**Comment by sysblade** [comment]
Hello, any update about this ? :)
